son-gtkapi/models/package_manager_service.rb

```diff
@@ -1,39 +1,41 @@
 ##
 ## Copyright (c) 2015 SONATA-NFV [, ANY ADDITIONAL AFFILIATION]
 ## ALL RIGHTS RESERVED.
-## 
+##
 ## Licensed under the Apache License, Version 2.0 (the "License");
 ## you may not use this file except in compliance with the License.
 ## You may obtain a copy of the License at
-## 
+##
 ##     http://www.apache.org/licenses/LICENSE-2.0
-## 
+##
 ## Unless required by applicable law or agreed to in writing, software
 ## distributed under the License is distributed on an "AS IS" BASIS,
 ## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 ## See the License for the specific language governing permissions and
 ## limitations under the License.
-## 
+##
 ## Neither the name of the SONATA-NFV [, ANY ADDITIONAL AFFILIATION]
-## nor the names of its contributors may be used to endorse or promote 
-## products derived from this software without specific prior written 
+## nor the names of its contributors may be used to endorse or promote
+## products derived from this software without specific prior written
 ## permission.
-## 
+##
 ## This work has been performed in the framework of the SONATA project,
-## funded by the European Commission under Grant number 671517 through 
-## the Horizon 2020 and 5G-PPP programmes. The authors would like to 
-## acknowledge the contributions of their colleagues of the SONATA 
+## funded by the European Commission under Grant number 671517 through
+## the Horizon 2020 and 5G-PPP programmes. The authors would like to
+## acknowledge the contributions of their colleagues of the SONATA
 ## partner consortium (www.sonata-nfv.eu).
 # encoding: utf-8
 require 'tempfile'
 require './models/manager_service.rb'
 
 class PackageManagerService < ManagerService
 
-  attr_reader :url, :logger
-  
   LOG_MESSAGE = 'GtkApi::' + self.name
-  
+
+  def self.url
+    @@url
+  end
+
```

Comment:
I've started using something like [` url = PackageManagerService.class_variable_get(:@@url)+'/admin/logs'`](https://github.com/sonata-nfv/son-gkeeper/blob/master/son-gtkapi/routes/package.rb#L137), instead of attribute accessors. Anyway I feel we'll have to review this piece of code, and make these 'service managers' instances of the `ManagerService` class, instead of subclasses of it like it is now.